interfaces/thirdparty/cp2k.py

```diff
@@ -382,27 +382,24 @@ def get_runscript(self):
             "popt",
             # parallel (MPI + OpenMP) general usage, threading might improve scalability and memory usage
             "psmp"}
-        suffix = Path(cp2k_command).suffix[1:]
-        if suffix not in available_executables:
-            msg = f"unrecognized cp2k executable: {cp2k_command}"
-            raise RuntimeError(msg)
-
-        # Try to run cp2k MPI binaries using mpirun and otherwise srun (if available)
-        if suffix in set({'sdbg', 'sopt'}):
-            ret = cp2k_command
-        else:
-            for command in ('mpirun', 'srun'):
-                try:
-                    subprocess.run([command, "--help"],
-                                   stdout=subprocess.DEVNULL)
-                    ret = f"{command} {cp2k_command}"
-                    break
-                except OSError:
-                    pass
 
-        ret += ' -i {} -o {}'.format(self._filename('inp'),
-                                     self._filename('out'))
-        return ret
+        available_mpi_commands = ('srun', 'mpirun')
+        mpi_command = ""
+
+        # If there is a MPI command the user knows what she is doing
+        if not any(mpi in cp2k_command for mpi in available_mpi_commands):
+            suffix = Path(cp2k_command).suffix[1:]
+
+            if suffix not in available_executables:
+                msg = f"unrecognized cp2k executable: {cp2k_command}"
+                raise RuntimeError(msg)
+
+            # Try to run cp2k MPI binaries using mpirun and otherwise srun (if available)
+            if suffix not in set({'sdbg', 'sopt'}):
+                available = (shutil.which(c) for c in available_mpi_commands)
+                mpi_command = next((c for c in available if c is not None), "")
```

Comment:
Even when `mpi_command = ""` then `cp2k_command` still be set to `"cp2k.popt"` by default (which requires `mpirun` or `srun` to properly function I believe?).

Reply:
Actually `cp2k.popt` does not strictly required the use of mpirun, these two command are equivalent:
```bash
mpirun -np 1 cp2k.popt -i input.inp -o output.out
cp2k.popt -i input.inp -o output.out
```

Reply:
Ah ok, nevermind then.